packages/core/src/telemetry/project-id.ts

```diff
@@ -0,0 +1,34 @@
+import { exec } from "child_process";
```

Comment:
I wonder if rather than using exec directly, we should use `execa` here so we don't need to do the promise-ification manually. (https://www.npmjs.com/package/execa?activeTab=readme).

Reply:
hmm - I think for something this simple we can skip adding a package and save those 77kb

Reply:
Agree we don't need `execa`! You know what - I'm backtracking 😅 , I actually think we should call this function `getGitRemoteUrl` and it should be in it's own file here called `git.ts` or `remote.ts`. I think what bothered me was calling it `project-id` because this file/fn doesn't actually generate the project ID, it just returns the raw git remote url (which is fine!).

Reply:
agree - got too biased by Nextjs implementation

Reply:
Looks good - should be able to remove this file now.